cv.typ

```diff
@@ -0,0 +1,70 @@
+#show heading: set text(font: "Linux Biolinum")
+
+#show link: underline
+#set page(
+ margin: (x: 0.9cm, y: 1.3cm),
+)
+#set par(justify: true)
+
+#let chiline() = {v(-3pt); line(length: 100%); v(-5pt)}
+
+= Eva Pace (she/her)
+
+#link("https://github.com/evaporei")[Github] | #link("https://gitlab.freedesktop.org/eva")[Gitlab] | #link("https://www.linkedin.com/in/evapace/")[LinkedIn] |
+#link("mailto:eba.pachi@gmail.com")[eba.pachi\@gmail.com]
+
+#chiline()
+
+Software Engineer with 6 years of experience. Worked with Web, FFI, WebAssembly and Blockchain. Currently looking for a low-level role in compilers or computer graphics.
+
+== Experience
+#chiline()
+
+*Rust Engineer* #h(1fr) 2021/04 -- 2023/02 \
+Edge & Node #h(1fr) San Francisco, US (remote)
+\
+I've worked to make #link("https://thegraph.com")[The Graph] move forward in the decentralization of public data. Delivered features such as: \
+- Integration of our first non-EVM compatible chain, NEAR Protocol;
+- Update of our AssemblyScript version, from the oldest to the newest (\~2 year change) that required a lot of ABI changes in our runtime code (WASM).
+Most of the code I made there is open source:
+- #link("https://github.com/graphprotocol/graph-node")[Rust node that indexes blockchain data and serves it via GraphQL];
+- #link("https://github.com/graphprotocol/graph-tooling")[JS CLI tool for creating subgraphs];
+- #link("https://github.com/graphprotocol/graph-ts")[AssemblyScript standard library].
+
+*Senior Software Engineer* #h(1fr) 2020/07 -- 2021/03 \
+Magazine Luiza #h(1fr) São Paulo, Brazil (remote) \
+I worked at MagaluPay, the company's digital account, that has over 2 million users. Took part in most of the new integration with the new government instant payments API (PIX). \
+
+*Software Engineer* #h(1fr) 2017/10 -- 2020/07 \
+Pagar.me #h(1fr) São Paulo, Brazil (on-site)
+- Developed multiple features at the core payments API;
```

Comment:
maybe include one cool/flashy feature, like `- Developed multiple features at the core payments API such as ...;`